src/datadoc/backend/datadoc_metadata.py

```diff
@@ -184,6 +187,16 @@ def extract_metadata_from_dataset(
             short_name=self.short_name,
             dataset_state=dapla_dataset_path_info.dataset_state,
             dataset_status=DatasetStatus.DRAFT,
+            assessment=Assessment.PROTECTED
```

Comment:
I think this section could be more readable if we implemented it as a mapping, similar to this example from Ruff: https://docs.astral.sh/ruff/rules/too-many-return-statements/#example

Reply:
Refactored unreadable code into a static method to increase readability